microsoft-edge/devtools-guide-chromium/whats-new/2022/04/devtools-101.md

```diff
@@ -0,0 +1,128 @@
+---
+title: What's new in DevTools (Microsoft Edge 101)
+description: Console tool in VS Code.  Source maps cached with IndexedDB.  Track pending network requests when exporting HAR files.  Updated UI and camera controls in 3D View.  Switch themes without reloading DevTools.  And more.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 04/28/2022
+---
+# What's New in DevTools (Microsoft Edge 101)
+
+The following sections list the announcements from the Microsoft Edge DevTools team.  Read these announcements, to try the latest features of Microsoft Edge DevTools and the Microsoft Edge DevTools extension for Microsoft Visual Studio and Visual Studio Code.
+
+To stay up to date with the latest and greatest features in your developer tools, download the [Microsoft Edge preview channels](https://www.microsoftedgeinsider.com/download).  The preview channels of Microsoft Edge (Beta, Dev, and Canary) give you access to the latest features of Microsoft Edge DevTools.  If you're on Windows, Linux, or macOS, consider using the Canary preview channel of Microsoft Edge as your default development browser.
+
+For the latest announcements, [follow the Microsoft Edge DevTools team on Twitter](https://twitter.com/EdgeDevTools).  To report a problem with DevTools or ask for a new feature, check out the new [MicrosoftEdge > DevTools](https://github.com/MicrosoftEdge/DevTools) repo.
+
+
+<!-- ====================================================================== -->
+## The Console tool is now available in Visual Studio Code
+
+<!-- Title: Visual Studio Code extension now with Console tool -->
+<!-- Subtitle: See log messages, run JavaScript, and use Console APIs right next to your code in Visual Studio Code. -->
+
+The Console is one of the most popular tools in DevTools.  You can now access the Console tool in the Microsoft Edge Developer Tools extension for Visual Studio Code.  This integrated DevTools Console allows you to see your log messages right next to your code.  Using the Console, you can also:
+*  Try out some JavaScript statements.
+*  Access the window object.
+*  Call the methods of the `Console` object, as described in [Console object API Reference](../../../console/api.md).
+*  Use the Console utilities to access the DOM, as described in [Console tool utility functions and selectors](../../../console/utilities.md).
+
+You can use the Console as the main tool, or use it together with the **Elements** and **Network** tools.
+
+The Console tool in the main toolbar of the Microsoft Edge DevTools extension for Visual Studio Code:
+
+![The Console tool in the main toolbar of the Microsoft Edge DevTools extension for Visual Studio Code.](devtools-101-images/console-tool-vsc-1.png)
```

Comment:
nice todo: revert my VS Code to default color theme, retake screenshots
@codepo8 - i might ask for your todo app